src/v/cluster/controller.cc

```diff
@@ -74,7 +74,10 @@ ss::future<> controller::wire_up() {
             config::shard_local_cfg().segment_fallocation_step.bind());
       })
       .then([this] { return _credentials.start(); })
-      .then([this] { return _authorizer.start(); })
+      .then([this] {
+          return _authorizer.start(
+            []() { return config::shard_local_cfg().superusers.bind(); });
```

Comment:
is this a lambda so that the value isn't constructed on core-0 and then copied to all other cores when the sharded<authorizer> is built? this seems like a pretty good pattern; it's nice to avoid having to initialize state through an `invoke_on_all` binding which I think is the most common pattern in the tree.

Reply:
Yep, exactly.  I considered creating some kind of "deferred binding" wrapper class that would take a property name and then generate a binding to the shard-local instance of it, but in practice these lambdas are generally one liners.